scripts/build-element.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-unused-vars */
```

Comment:
.esm.js file shouldn't use imports, it must be bundled with rollup as well. Actually its not needed, keep just .esm.browser only file and rename it to just `.esm`. So we should have only two files atropos-element.js and atropos-element.esm.js, and both of them should have minified versions